src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/RelationToMultiPolygonCommand.java

```diff
@@ -0,0 +1,195 @@
+{
+    // maximum WKT string length for terminal printing
+    private static final int MAX_LENGTH = 1024 * 150;
+    private static final String OUTPUT_ATLAS_FORMAT = "%d.atlas";
+    private static final String OUTPUT_WKT_FORMAT = "%d.wkt";
+
+    private static final String RELATION_ID_OPTION_LONG = "id";
+    private static final String RELATION_ID_OPTION_DESCRIPTION = "A relation ID to build a multipolygon for";
+    private static final String RELATION_ID_OPTION_HINT = "id";
+
+    private static final String OUTPUT_WKT_OPTION_LONG = "wkt";
+    private static final String OUTPUT_WKT_OPTION_DESCRIPTION = "Only output WKT file";
+
+    private final OptionAndArgumentDelegate optionAndArgumentDelegate;
+    private final CommandOutputDelegate outputDelegate;
+
+    private long relationId;
+    private final Set<Atlas> relationAtlases = new HashSet<>();
+
+    public static void main(final String[] args)
+    {
+        new RelationToMultiPolygonCommand().runSubcommandAndExit(args);
+    }
+
+    public RelationToMultiPolygonCommand()
+    {
+        super();
+        this.optionAndArgumentDelegate = this.getOptionAndArgumentDelegate();
+        this.outputDelegate = this.getCommandOutputDelegate();
+    }
+
+    @Override
+    public String getCommandName()
+    {
+        return "relation2multipolygon";
+    }
+
+    @Override
+    public String getSimpleDescription()
+    {
+        return "build a multipolygon from a relation spanning a given set of atlases";
+    }
+
+    @Override
+    public void registerManualPageSections()
+    {
+        addManualPageSection("DESCRIPTION", RelationToMultiPolygonCommand.class
+                .getResourceAsStream("RelationToMultiPolygonDescriptionSection.txt"));
+        addManualPageSection("EXAMPLES", RelationToMultiPolygonCommand.class
+                .getResourceAsStream("RelationToMultiPolygonCommandExamplesSection.txt"));
+        super.registerManualPageSections();
+    }
+
+    @Override
+    public void registerOptionsAndArguments()
+    {
+        registerOptionWithRequiredArgument(RELATION_ID_OPTION_LONG, RELATION_ID_OPTION_DESCRIPTION,
+                OptionOptionality.REQUIRED, RELATION_ID_OPTION_HINT);
+        registerOption(OUTPUT_WKT_OPTION_LONG, OUTPUT_WKT_OPTION_DESCRIPTION,
+                OptionOptionality.OPTIONAL);
+        super.registerOptionsAndArguments();
+    }
+
+    @Override
+    public int start()
+    {
+        this.relationId = this.optionAndArgumentDelegate
+                .getOptionArgument(RELATION_ID_OPTION_LONG, Long::valueOf).orElse(-1L);
+
+        if (this.relationId == -1L)
+        {
+            this.outputDelegate.printlnErrorMessage("invalid relation identifier!");
+            return -1;
+        }
+        return 0;
+    }
+
+    @Override
+    protected int finish()
+    {
+        if (this.relationAtlases.isEmpty())
+        {
+            this.outputDelegate.printlnErrorMessage("no atlases found with relation");
+            return 1;
```

Comment:
Any reason for returning 1 here, and -1 above? As a note: -1 will likely show up as 255 in most shells. Maybe include a note about the different return codes in the description? I know this is a nit :P

Reply:
oh whoops, this should be -1